src/resources/js/tribe-events-bar.js

```diff
@@ -24,6 +24,7 @@ var tribe_events_bar_action;
 	 */
 
 	$( document ).ready( function() {
+		$('select').selectability();
```

Comment:
Rather than initializing _all_ `<select>` elements, this should be targeted at the boxes we wish to change. In this case, only the tribe bar selectbox.